tests/helpers/docker.bash

```diff
@@ -12,6 +12,10 @@ function wait_for_dockerd {
   retry_run 10 1 "docker ps"
 }
 
+function wait_for_dockerd {
```

Comment:
Looks like an error here (we are repeating an existing function).

Reply:
Interesting, not sure how that happened. Will generate a new patch once the CI concludes.

Reply:
Done.